utils/nano/Makefile

```diff
@@ -22,18 +22,31 @@ PKG_CPE_ID:=cpe:/a:gnu:nano
 
 PKG_INSTALL:=1
 PKG_BUILD_PARALLEL:=1
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(BUILD_VARIANT)/$(PKG_NAME)-$(PKG_VERSION)
```

Comment:
wasn't there an upstream commit that already does this?

Reply:
you are apparently right. Since 2019 there is a default line for that purpose. 
I will modify and test.

Reply:
Is it possible to include unicode support in that plus variant?